modules/oadp-backup-restore-cr-issues.adoc

```diff
@@ -75,13 +75,78 @@ time="2023-02-17T16:33:13Z" level=error msg="Error backing up item" backup=opens
 $ oc delete backup <backup> -n openshift-adp
 ----
 
-. If required, clean up the stored data on the `BackupStorageLocation` to free up space.
+. If required, free up space by cleaning up the stored data on the `BackupStorageLocation`.
 
-. Apply label `velero.io/csi-volumesnapshot-class=true` to the `VolumeSnapshotClass` object:
+. Apply the label `velero.io/csi-volumesnapshot-class=true` to the `VolumeSnapshotClass` object:
 +
 [source,terminal]
 ----
 $ oc label volumesnapshotclass/<snapclass_name> velero.io/csi-volumesnapshot-class=true
 ----
 
-. Create a new `Backup` CR.
\ No newline at end of file
+. Create a new `Backup` CR.
+
+[id="datamover-backup-cleanup_{context}"]
+== Datamover backup cleanup
+
+[NOTE]
+====
+Applies to OADP 1.1.x and newer.
```

Comment:
Applies to only 1.1.x.

Reply:
s/Applies to OADP 1.1.x and newer/Applies to OADP 1.1.x/g

Reply:
Remover ".x and newer" from line 94